src/webui/api/appcontroller.cpp

```diff
@@ -649,8 +649,22 @@ void AppController::setPreferencesAction()
     }
     // Current network interface address
     if (hasKey("current_interface_address")) {
-        const QHostAddress ifaceAddress {it.value().toString().trimmed()};
-        session->setNetworkInterfaceAddress(ifaceAddress.isNull() ? QString {} : ifaceAddress.toString());
+        const QString value = it.value().toString().trimmed();
+        bool ok = false;
+        uint option = value.toUInt(&ok);
```

Comment:
IMO you should always use `int` unless doing bits operations.

Reply:
And add `const`.

Reply:
Damn it. I forgot about const because I was trying something else initially.